core.py

```diff
@@ -250,7 +255,7 @@ def log_consumers(consumers):
         logger.info(f'PID {pid}: {info}')
 
 
-def unmount(name):
+def unmount(name, is_schain=True):
```

Comment:
I think we should also expose is_schain parameter in the api interface in app.py file